### DISCUSSION TITLE
Synology Container Manager cant authenticate

### DISCUSSION THREAD

**Discussion by LastTenth** [mrlt8/docker-wyze-bridge#1039] [Q&A] [answered]
I'm new to docker and I tried to setup my Wyze Cam. I have set my env to include my login and password, as well as opened the ports on my NAS's firewall. But I repeatedly get these errors...

2023/11/09 01:07:59 | stdout | [WyzeBridge] 👋 goodbye!
2023/11/09 01:07:59 | stdout | [WyzeBridge] Stopping 0 streams
2023/11/09 01:07:59 | stdout | [WyzeBridge] [API] ERROR: HTTPSConnectionPool(host='auth-prod.api.wyze.com', port=443): Max retries exceeded with url: /v3/user/login (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f32eadaa550>: Failed to resolve 'auth-prod.api.wyze.com' ([Errno -3] Temporary failure in name resolution)"))
2023/11/09 01:07:39 | stdout | [WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...
2023/11/09 01:07:39 | stdout | [WyzeBridge] 🔍 Could not find local cache for 'auth'
2023/11/09 01:07:39 | stdout | [WyzeBridge] ☁️ Fetching 'cameras' from the Wyze API...
2023/11/09 01:07:39 | stdout | [WyzeBridge] 🔍 Could not find local cache for 'cameras'
2023/11/09 01:07:39 | stdout | [WyzeBridge] [API] ERROR: HTTPSConnectionPool(host='auth-prod.api.wyze.com', port=443): Max retries exceeded with url: /v3/user/login (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f32eada7d50>: Failed to resolve 'auth-prod.api.wyze.com' ([Errno -3] Temporary failure in name resolution)"))
2023/11/09 01:07:19 | stdout | [WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...
2023/11/09 01:07:19 | stdout | [WyzeBridge] 🔍 Could not find local cache for 'auth'
2023/11/09 01:07:19 | stdout | [WyzeBridge] ☁️ Fetching 'user' from the Wyze API...
2023/11/09 01:07:19 | stdout | [WyzeBridge] 🔍 Could not find local cache for 'user'
2023/11/09 01:07:19 | stdout | [WyzeBridge] [API] ERROR: HTTPSConnectionPool(host='auth-prod.api.wyze.com', port=443): Max retries exceeded with url: /v3/user/login (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f32ece99e50>: Failed to resolve 'auth-prod.api.wyze.com' ([Errno -3] Temporary failure in name resolution)"))
2023/11/09 01:06:59 | stdout | [WyzeBridge] [33mPress CTRL+C to quit[0m
2023/11/09 01:06:59 | stdout | * Running on http://172.17.0.2:5000
2023/11/09 01:06:59 | stdout | * Running on http://127.0.0.1:5000
2023/11/09 01:06:59 | stdout | * Running on all addresses (0.0.0.0)
2023/11/09 01:06:59 | stdout | [WyzeBridge] [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
2023/11/09 01:06:59 | stdout | * Debug mode: off
2023/11/09 01:06:59 | stdout | * Serving Flask app 'frontend'
2023/11/09 01:06:59 | stdout | [WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...
2023/11/09 01:06:59 | stdout | [WyzeBridge] 🔍 Could not find local cache for 'auth'
2023/11/09 01:06:59 | stdout |  
2023/11/09 01:06:59 | stdout | 🚀 DOCKER-WYZE-BRIDGE v2.5.0

I spent the last couple hours googling, saw something about adding and IP but that was for another app and I can't find out to do it on Container Manager. Can anyone help me out?

**Comment by mrlt8** [comment]
Sounds like a DNS issue? See https://github.com/mrlt8/docker-wyze-bridge/issues/721#issuecomment-1454083576

**Reply by LastTenth** [reply]
Yeah I saw that thread. I can’t figure out how to fix that, or perform steps similar to those, though.

**Reply by mrlt8** [reply]
I believe you'd need to update the AdGuard/pihole container and include the IP when exposing the DNS ports for so that the container running the bridge can connect.

**Reply by LastTenth** [reply]
I don’t have those. It’s something I need to make the wyze bridge work?

**Reply by mrlt8** [reply]
No, they're not needed, but seem to be a common cause for the issue.

You may need to look into the DNS settings on the Synology since that's where the container is trying to copy the DNS servers from.

**Reply by LastTenth** [reply]
The DNS on my synology is ok I think. It has no connection problems in/out of the network. It seems to be just the container and I can’t figure out how to tell it to use the synology’s settings.

**Reply by HussX** [reply]
Add me to this list for now. I'm migrating from my pi to Syno and ended at the same errors. Digging in, now.

**Comment by HussX** [comment, marked as answer]
Found it. Wyze bridge makes a custom docker IP range, mine was 172.18.0.0/16. Allow it in your firewall rules on the Synology (before block all rule) and you should be good. Mine works!
